pkg/webservice/state_dump.go

```diff
@@ -157,29 +160,48 @@ func doStateDump(w io.Writer) error {
 	if err != nil {
 		return err
 	}
-	if _, err = w.Write(prettyJSON); err != nil {
+
+	stateLog := log.New(w, "", 0)
+	stateDumpFilePath := getStateDumpFilePath()
+	stateLog.SetOutput(&lumberjack.Logger{
+		Filename:   stateDumpFilePath,
+		MaxSize:    10,
+		MaxBackups: 10,
```

Comment:
shouldn't these two be configurable as well?

Reply:
I'd like to leave as hardcoded for now if that's ok and file a follow-up JIRA to make them configurable.

Reply:
Fine, OK for me.